Vagrantfile

```
@@ -0,0 +1,34 @@
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL 
+    sudo apt-get -y update
+    #TODO: Install pyenvv prerequisites
```

Can get rid of the "TODO" now 🙂 
```suggestion
    # Install pyenvv prerequisites
```